src/applications/vaos/new-appointment/components/ProviderSelectPage/ScheduleWithDifferentProvider.jsx

```diff
@@ -1,13 +1,30 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { useHistory } from 'react-router-dom';
+import { useDispatch, useSelector } from 'react-redux';
 import { getFacilityPhone } from '../../../services/location';
 import FacilityPhone from '../../../components/FacilityPhone';
+import getNewAppointmentFlow from '../../newAppointmentFlow';
+import { startRequestAppointmentFlow } from '../../redux/actions';
 import { ELIGIBILITY_REASONS } from '../../../utils/constants';
 
+function handleClick(history, dispatch, requestDateTime) {
+  return e => {
+    // Stop default behavior for anchor tag since we are using React routing.
```

Comment:
Hmm I'm curious if this is strictly necessary. Would something like [this](https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/applications/vaos/appointment-list/components/ScheduleAppointmentLink.jsx#L18) not work? This is obviously not blocking but I'm curious.

Reply:
Thanks, I can see now that a simple link like I suggested won't work but I don't think what we have here is sufficient either since history/data isn't preserved. For example, wouldn't this break the "Back" button and breadcrumb from the request page be broken?